cli/index.js

```diff
@@ -290,17 +290,17 @@ function run(paths) {
     expandFiles(paths).map(filePath => {
       const output = args.output;
       return accessFile(filePath, fs.constants.R_OK)
-        .catch(err => {
-          console.error(err.toString());
-          process.exit(1);
-        })
         .then(() => {
           return buildHtmlAndJson({
             filePath,
             output,
             buildHtml: args["build-html"],
             quiet: args["quiet"]
           });
+        })
+        .catch(ex => {
+          console.error(ex);
```

Comment:
why not log the error at the highest level (i.e. when catching run) and omit this skip call?

Reply:
I'm not sure I understand. This catches any error within `buildHtmlAndJson` in general. Now, *inside* `buildHtmlAndJson` it might do its own error handling of individual stuff.